### PR TITLE
surface errors from image wait

### PR DESCRIPTION
**Pull request by tomkennedy513**
- only display error message if it exists to prevent dangling colon in message

### CODE REVIEW

File: pkg/logs/wait_for_image.go [hunk 1]
```diff
@@ -138,8 +142,12 @@ func (w *imageWaiter) waitBuild(ctx context.Context, writer io.Writer, namespace
 		return "", errors.New("unexpected object received")
 	}
 
-	if build.Status.GetCondition(corev1alpha1.ConditionSucceeded).IsFalse() {
-		return "", errors.Errorf("update to image failed")
+	if condition:= build.Status.GetCondition(corev1alpha1.ConditionSucceeded); condition.IsFalse() {
+		if condition.Message != "" {
```
Comment (sukhil-suresh) [inline pkg/logs/wait_for_image.go:146]: wonder if a single `return "", errors.Wrap(errors.New(condition.Message), "update to image failed")` without the `condition.Message != ""`  check is sufficient? not sure.

Comment (tomkennedy513) [reply pkg/logs/wait_for_image.go:146]: it will try to wrap still so you get "update to image failed: " with the dangling colon

Comment (sukhil-suresh) [reply pkg/logs/wait_for_image.go:146]: got it. thanks for clarifying

File: pkg/logs/wait_for_image.go [hunk 2]
```diff
@@ -56,7 +56,11 @@ func (w *imageWaiter) resultOfImageWait(ctx context.Context, writer io.Writer, g
 		return w.waitBuild(ctx, writer, image.Namespace, image.Status.LatestBuildRef)
 	}
 
-	if image.Status.GetCondition(corev1alpha1.ConditionReady).IsFalse() {
+	if condition := image.Status.GetCondition(corev1alpha1.ConditionReady); condition.IsFalse() {
+		if condition.Message != "" {
+			return "", errors.Wrap(errors.New(condition.Message), fmt.Sprintf("update to image %s failed", image.Name))
```
Comment (matthewmcnew) [inline pkg/logs/wait_for_image.go:61]: Could we use a single `errors.Errorf` here?

**Review by matthewmcnew — APPROVED** [review]

**Review by sukhil-suresh — APPROVED** [review]

File: pkg/logs/wait_for_image.go (hunk 2 above)
Comment (tomkennedy513) [reply pkg/logs/wait_for_image.go:61]: ya I can change that

**Review by djoyahoy — APPROVED** [review]

**Pull request merged**